### PR TITLE
Hs 8 student dashboard

### PR DESCRIPTION
**Pull request by mitrasingh**
**Main Student Dashboard**

This page is the home page for the student. They can view the status of their project from here at a high level, they can navigate to the Student Credit Detail Page from here, they can also generate a transcript for this page.
As a student user, I need to be able to see and action the list of credits I am working on and have completed so that I can manage my transcript.

View and Download button (bottom corner) are not included as we are holding off as per Elizabeth request.

### CODE REVIEW

**Comment by mitrasingh** [comment]
Ready for review @thedomcab

File: package.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
     "html-react-parser": "^5.1.10",
     "next": "13.5.6",
     "react": "^18",
-    "react-dom": "^18"
+    "react-dom": "^18",
+    "react-property": "^2.0.2"
```
Comment (thedomcab) [inline package.json:18]: What is this being used for?

File: src/app/students/dashboard/DeleteButton.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+const DeleteButton = () => {
+	return (
+		<button>
+			<svg
+				xmlns="http://www.w3.org/2000/svg"
+				fill="none"
+				viewBox="0 0 24 24"
+				strokeWidth={3}
+				stroke="#D40000"
```
Comment (thedomcab) [inline src/app/students/dashboard/DeleteButton.tsx:9]: Could you convert these html attributes to use CSS

File: src/app/students/dashboard/TakeActionButton.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const TakeActionButton = () => {
+	return (
+		<button
+			className={`h-[1.5rem] px-4 bg-[#805DBE] text-white text-[.75rem] font-normal hover:bg-violet-700`}
```
Comment (thedomcab) [inline src/app/students/dashboard/TakeActionButton.tsx:4]: could you use padding instead of height?

File: package.json (hunk 1 above)
Comment (mitrasingh) [reply package.json:18]: resolved a conflict with main by installed react-property

File: src/app/students/dashboard/WelcomeBanner.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+const WelcomeBanner = () => {
+	return (
+		<section className="h-[10.9375rem] bg-[#805DBE] rounded-xl pb-14 mb-6 mx-12">
```
Comment (thedomcab) [inline src/app/students/dashboard/WelcomeBanner.tsx:3]: avoid using height and use padding for this as well.

**Comment by mitrasingh** [comment]
@thedomcab resolved all issues, keep me posted if any other changes are needed for this PR.

**Review by thedomcab — APPROVED** [review]

**Pull request merged**